common-pg/src/main/scala/com/socrata/pg/query/QueryServerHelper.scala

```diff
@@ -252,7 +252,7 @@ object QueryServerHelper {
   val typeReps: Map[SoQLType, SqlColumnRep[SoQLType, SoQLValue]] = {
     // build fake columnInfo where most properties values do not matter except type
     val cid = new ColumnId(0)
-    val datasetInfo = DatasetInfo(DatasetId.Invalid, 0, 0, "", Array.empty[Byte], None)(null)
+    val datasetInfo = DatasetInfo(DatasetId.Invalid, 0, 0, "", Array.empty[Byte], DatasetResourceName(""))(null)
```

Comment:
Am instinctively kinda heebie-jeebie about using an empty string here, but eh.

Reply:
Yeah, I'm not sure what it thinks it's doing here (it's definitely not _creating_ anything in the database, the queryserver doesn't have rights to do that) but it's OA anyway..